src/os/style/strokereader.js

```diff
@@ -31,8 +31,13 @@ os.style.StrokeReader.prototype.getOrCreateStyle = function(config) {
 
   hash = 31 * hash + goog.string.hashCode(color) >>> 0;
 
+  var dash = /** @type {!Array<number>} */ (config['lineDash']) || os.style.DEFAULT_LINE_STYLE;
+
+  hash += goog.string.hashCode(dash.join());
```

Comment:
Please use the same method of updating the hash as other config components:
```
hash = 31 * hash + goog.string.hashCode(dash.join()) >>> 0;
```
This method avoids style collisions by isolating each component added to the hash.

WRT later comments, only change the hash if `lineDash` is defined. The default should be `undefined`.